### ISSUE TITLE
Ubuntu 22.04 

### ISSUE THREAD

**Issue by gunyakov** [systemed/tilemaker#666]
When I install latest version from source in ubuntu 22.04 by commands
`
`
tilemaker bulgaria-latest.osm.pbf bulgaria-latest.mbtiles
Couldn't open .json config: config.json
`

**Comment by systemed** [comment]
Yes, tilemaker doesn't install data files such as config.json or process.lua to a static location (and nor does it claim anywhere that it does!).

I'd be happy to receive a PR for that, but it's not really my area of expertise and therefore not something I'm planning to work on.

I'll close #667 in favour of this as it's essentially the same issue.
